### PR TITLE
Added automation for No Result Screen

### PR DESCRIPTION
**Pull request by SyedaGinii**
Ui Automation for No Result Screen
Jira ticket: https://ginis.atlassian.net/browse/PM-28

### CODE REVIEW

File: bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/NoResultScreen.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+
+class NoResultScreen {
```
Comment (danicretu) [inline bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/NoResultScreen.kt:9]: [Request] Please update the methods in this class to match the changes you made for [this](https://github.com/gini/gini-mobile-android/pull/565) PR. 
By that I mean returning the result of the check to be used in the main test case.

File: bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    private lateinit var idlingResource: SimpleIdlingResource
+
+    @Before
+    fun setup() {
+        idlingResource = SimpleIdlingResource(5000)
+        IdlingRegistry.getInstance().register(idlingResource)
+    }
+
+    @Test
+    fun test1_uploadInvalidImageAndClickEnterManuallyButton_NavigatesToMainScreen() {
```
Comment (danicretu) [inline bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt:48]: [Request] Please update this method to make an assertion at the end, like you did in [this](https://github.com/gini/gini-mobile-android/pull/565) PR.

File: bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        noResultScreen.checkMultiPagesTitleIsDisplay()
+        idlingResource.waitForIdle()
+        noResultScreen.checkEnterManuallyButtonIsDisplay()
+        noResultScreen.clickEnterManuallyButton()
+        mainScreen.assertDescriptionTitle()
+        idlingResource.waitForIdle()
+    }
+
+    @Test
+    fun test2_uploadInvalidPdfAndClickEnterManuallyButton_NavigatesToMainScreen() {
```
Comment (danicretu) [inline bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt:73]: [Request] Same request as above.

**Review by danicretu — CHANGES_REQUESTED** [review]
Good job Syeda! 👌 I added some requests to match the result of our previous discussions, and the improvements you made in the other PR.

File: bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt (hunk 3 above)
Comment (SyedaGinii) [reply bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/NoResultTests.kt:73]: The assertion at the end of test-case is added.
```
val isDescriptionTitleVisible = mainScreen.assertDescriptionTitle()
        assertEquals(true, isDescriptionTitleVisible)
```

**Review by danicretu — APPROVED** [review]
Thank you Syeda, the test cases look great! 🥳👌

**Pull request merged**